.circleci/config.yml

```diff
@@ -62,6 +62,28 @@ jobs:
           paths:
             - ./coverage
 
+  type_check:
+    docker:
+      - image: circleci/node:12
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps1-{{ checksum ".circleci/config.yml" }}-{{ checksum "yarn.lock" }}
+      
+      - run:
+          name: Install packages
+          command: yarn install
+      
+      - save_cache:
+          key: deps1-{{ checksum ".circleci/config.yml" }}-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+
+      - run:
+          name: Check typings
+          command: yarn tsc --noEmit
```

Comment:
can we have this as a script in the `package.json` that we also can execute it locally! 🙂

Reply:
ups, vergessen:)